README.md

```diff
@@ -36,7 +37,7 @@ source install/local_setup.sh
 
 Run the noop demo
 ```bash
-ros2 run ros_sec_test runner __params:=`ros2 pkg prefix ros_sec_test`/share/ros_sec_test/examples/params.yaml
+ros2 run ros_sec_test ros_sec_test __params:=`ros2 pkg prefix ros_sec_test`/share/ros_sec_test/examples/params.yaml
```

Comment:
This seems like an unrelated change - would you mind submitting this in another PR (the doc + renaming the binary)?

Reply:
ok!